backend/fms_core/templates.py

```diff
@@ -120,6 +121,29 @@
       ("Library", "Sample Container Coordinates", "coordinates"),],
 }
 
+LIBRARY_QC_TEMPLATE = {
+  "identity": {"description": "Template to add library QC", "file": static("submission_templates/Library_QC_v3_9_0.xlsx")},
+  "sheets info": [
+      {
+        'name': 'LibraryQC',
+        'headers': ['Library Name', 'Library Container Barcode', 'Library Container Coord', 'Initial Volume (uL)',
+                    'Measured Volume (uL)', 'Volume Used (uL)', 'Strandedness', 'Library size (bp)', 'Concentration (nM)',
+                    'Concentration (ng/uL)', 'NA Quantity (ng)', 'Quality Instrument', 'Quality Flag',
+                    'Quantity Instrument', 'Quantity Flag', 'QC Date (YYYY-MM-DD)', 'Comment'],
+      },
+  ],
+  "prefill info": [
+      # TODO queryset columns
+      ("LibraryQC", "Library Name", "?"),
+      ("LibraryQC", "Library Container Barcode", "?"),
+      ("LibraryQC", "Library Container Coord", "?"),
+      ("LibraryQC", "Initial Volume (uL)", "?"),
+      ("LibraryQC", "Strandedness", "?"),
```

Comment:
that would be more complicated i will have to think this one over ... The path would be "derived_sample_not_pool__library__strandedness" but the pooling thing may interfere So we may need to make a special accessor to this and the library_size which is also behind a many-to-many relationship.